src/components/common/Like.tsx

```diff
@@ -0,0 +1,36 @@
+import { Button, ButtonProps, Text } from '@chakra-ui/react';
+import { CheckIcon } from '@chakra-ui/icons';
+import { useState } from 'react';
+
+type LikeProps = ButtonProps & {
+  likeCount: number;
+};
+
+const Like = ({ likeCount, ...props }: LikeProps) => {
+  const [count, setCount] = useState(likeCount);
+  const [like, setLike] = useState(false);
+  const [click, setClick] = useState(false);
+
+  const handleToggle = () => {
+    if (click) return;
+    setClick(true);
+    if (like) {
+      // 비동기 통신
+      setCount((prev: number) => prev - 1);
+    } else {
+      // 비동기 통신
+      setCount((prev: number) => prev + 1);
+    }
+    setLike((prev: boolean) => !prev);
+    setClick(false);
+  };
+
+  return (
+    <Button {...props} onClick={handleToggle}>
+      <CheckIcon color={like ? 'yellow' : 'black'} marginRight="2" />
```

Comment:
```suggestion
      <CheckIcon color={like ? 'yellow' : 'black'} mr={2} />
```

이런 식으로 `mr`로 바꾸는게 좋을 것 같아요!